locales/en/index.yml

```diff
@@ -178,6 +179,10 @@ global:
     spidLogin:
       title: Remember to login to IO!
       message: Login with your digital identity on IO, the public services app
+  media:
+    email: "Email"
+    pec: "Certified email"
+    phone: "Phone"
```

Comment:
@thisisjp @Undermaken this part is opinionated.
I've explained the rationale in the description but please feel free to come up with different proposals.